ionic2/app/pages/map/map.page.html

```diff
@@ -10,4 +10,7 @@
   </button>
 
   <poke-poi-card #poiCard></poke-poi-card>
+
+  <button id="artificial-mob-button"
```

Comment:
I'm glad you added this button so I can test your functionality. But obviously that shouldn't end up in the final product. So I think either the button should be removed at all or somehow hidden such that it can only by used by developers.